source/docs/software/advanced-controls/introduction/tuning-vertical-arm.rst

```diff
@@ -54,66 +65,119 @@ Simulation - Vertical Arm
       </script>
     </div>
 
-Arm Tuning Step 1: Feedback-Only
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. note:: When "increasing" a value, multiply it by two until the expected effect is observed. Similarly, when "decreasing" a value, divide by two (this is called an "exponential search", and is very good for finding positive constants when we don't have a good idea of how large they should be). Once you find the point where the expected effect starts or stops, switch to "bumping" the value up and down by ~10% until the behavior seems to stop improving.
```

Comment:
It's hard to make it its own sentence without restructuring the entire note, because the note goes on to give differing instructions immediately after and so the flow of the primary gist of the note is interrupted if you do that.  I think people are likely to read the parenthetical, so they'll get the vocab even if it's in parens.